src/Database/PostgreSQL/PQTypes/Checks.hs

```diff
@@ -508,7 +508,7 @@ checkDBStructure options tables = fmap mconcat . forM tables $ \(table, version)
       runQuery_ $ sqlGetForeignKeys table
       fkeys <- fetchMany fetchForeignKey
       triggers <- getDBTriggers tblName
-      checkedOverlaps <- checkOverlappingIndexes
+      checkedOverlaps <- checkOverlappingIndexes (mkSQL . T.pack $ tblNameString table)
```

Comment:
```suggestion
      checkedOverlaps <- checkOverlappingIndexes (tblName table)
```

And pass `RawSQL ()` around and inside `checkOverlappingIndexesQuery` you can use `raw` function to convert from `RawSQL ()` to `SQL`. AFAIR this is the general theme of how using RawSQL in constructing SQLs throughout the library is done, so let's be consistent.